Server/tests/test_manage_profiler.py

```
@@ -0,0 +1,296 @@
+
+def test_get_counters_omits_none_counters(mock_unity):
+    result = asyncio.run(
+        manage_profiler(SimpleNamespace(), action="get_counters", category="Memory")
+    )
+    assert result["success"] is True
+    assert "counters" not in mock_unity["params"]
+
+
+def test_profiler_start_forwards_log_file(mock_unity):
+    result = asyncio.run(
+        manage_profiler(SimpleNamespace(), action="profiler_start", log_file="/tmp/profile.raw")
```

Ruff (0.15.7)

[error] Probable insecure usage of temporary file or directory: "/tmp/profile.raw"

(S108)